src/main/resources/locales/en/translation/steps-to-making-your-claim.json

```diff
@@ -1,10 +1,26 @@
 {
-  "title": "Steps to making your claim",
```

Comment:
You need title translation. Please look at the tab name.

Reply:
this has been addressed